### DISCUSSION TITLE
フォロー中のタグが表示されていない

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#579] [要望・アイデア・不具合]
Pythonのタグをフォローしているのですが、添付の通りにマイページのフォロー中のタグに表示されていません。
また、ホームとタイムラインにも影響していると思われます。

サーバのレスポンスを見ると、レスポンスにも含まれていないように見えます。

ご確認お願いします。

<img width="362" alt="スクリーンショット 2023-12-06 13 16 12" src="https://github.com/increments/qiita-discussions/assets/38848514/b238118c-d171-479a-bffd-04d2484f2b33">
<img width="372" alt="スクリーンショット 2023-12-06 13 17 00" src="https://github.com/increments/qiita-discussions/assets/38848514/bc7c98a8-c769-4af6-95a0-3413c4e7895e">

**Comment by tanaka-asami415** [comment]
@Brutus03 
お問い合わせありがとうございます。

マイページのサイドバーには20件までしか表示されない仕様となっているためですが
その事が分かりにくいため、20件より多い場合に分かりやすい見た目にしたいと思います🙇

画像の箇所をクリックしていただくと表示される、フォローしているタグページ一覧の3ページ目にはpytohnが表示されます。

![image](https://github.com/increments/qiita-discussions/assets/103916185/066021c4-eed0-4634-8953-c0378b2176b9)

確認いたしましたが、ホームとタイムラインには影響はしていないようです🙏

**Reply by Brutus03** [reply]
> その事が分かりにくいため、20件より多い場合に分かりやすい見た目にしたいと思います🙇

仕様について承知しました。あと、表示されているフォロー中のタグは、フォローしているタグの降順ということも分かるとより良いと思います。

**Reply by tanaka-asami415** [reply]
@Brutus03 
お待たせいたしました！

フォロー中の数が20件より多い場合に分かりやすい見た目に変更いたしました！
https://qiita.com/release-notes/2024-01-09

> 表示されているフォロー中のタグは、フォローしているタグの降順ということも分かるとより良いと思います。

上記確認致しましたが、最近フォローした順に表示される仕様となっております。
この点はヘルプページに記載をしようと思います🙏

**Reply by Brutus03** [reply]
上記ご対応ありがとうございました。
以前より分かりやすくなったと思いますので本件、クローズで大丈夫です。
